### PR TITLE
list of projects using sphinx-gallery

### LINKED ISSUE #536: Mention list of projects using sphinx-gallery in a documentation page
At the moment it's only in the README on Github, so the information is a bit lost for users (as opposed to contributors). Maybe in the FAQ? Or a separate page in "advanced usage and information", If you think it's a good idea I can take care of this.

### PR DESCRIPTION
**Pull request by emmanuelle**
closes #536

### CODE REVIEW

File: doc/projects_list.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* `PyStruct <https://pystruct.github.io/auto_examples/index.html>`_
+* `GIMLi <http://www.pygimli.org/_examples_auto/index.html>`_
+* `Nestle <https://kbarbary.github.io/nestle/examples/index.html>`_
+* `pyRiemann <https://pythonhosted.org/pyriemann/auto_examples/index.html>`_
+* `scikit-image <http://scikit-image.org/docs/dev/auto_examples/>`_
+* `Astropy <http://docs.astropy.org/en/stable/generated/examples/index.html>`_
+* `SunPy <http://docs.sunpy.org/en/stable/generated/gallery/index.html>`_
+* `PySurfer <https://pysurfer.github.io/>`_
+* `Matplotlib <https://matplotlib.org/index.html>`_ `Examples <https://matplotlib.org/gallery/index.html>`_ and `Tutorials  <https://matplotlib.org/tutorials/index.html>`__
+* `PyTorch tutorials <http://pytorch.org/tutorials>`_
```
Comment (choldgraf) [inline doc/projects_list.rst]: Are we sure this is SG? I looked at it and it doesn't seem the same CSS, but maybe they modified it?

File: doc/projects_list.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Here is a list of projects using `sphinx-gallery`.
+
+* `Sphinx-Gallery <https://sphinx-gallery.github.io/en/latest/auto_examples/index.html>`_
+* `Scikit-learn <http://scikit-learn.org/dev/auto_examples/index.html>`_
+* `Nilearn <https://nilearn.github.io/auto_examples/index.html>`_
+* `MNE-python <https://www.martinos.org/mne/stable/auto_examples/index.html>`_
+* `PyStruct <https://pystruct.github.io/auto_examples/index.html>`_
+* `GIMLi <http://www.pygimli.org/_examples_auto/index.html>`_
+* `Nestle <https://kbarbary.github.io/nestle/examples/index.html>`_
+* `pyRiemann <https://pythonhosted.org/pyriemann/auto_examples/index.html>`_
```
Comment (choldgraf) [inline doc/projects_list.rst]: this one didn't work for me :-/

File: doc/projects_list.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+* `scikit-image <http://scikit-image.org/docs/dev/auto_examples/>`_
+* `Astropy <http://docs.astropy.org/en/stable/generated/examples/index.html>`_
+* `SunPy <http://docs.sunpy.org/en/stable/generated/gallery/index.html>`_
+* `PySurfer <https://pysurfer.github.io/>`_
+* `Matplotlib <https://matplotlib.org/index.html>`_ `Examples <https://matplotlib.org/gallery/index.html>`_ and `Tutorials  <https://matplotlib.org/tutorials/index.html>`__
+* `PyTorch tutorials <http://pytorch.org/tutorials>`_
+* `Cartopy <http://scitools.org.uk/cartopy/docs/latest/gallery/>`_
+* `PyVista <https://docs.pyvista.org/examples/>`_
+* `SimPEG <http://docs.simpeg.xyz/content/examples/>`_
+* `PlasmaPy <http://docs.plasmapy.org/en/master/auto_examples/>`_
```
Comment (choldgraf) [inline doc/projects_list.rst]:
```suggestion
* `PlasmaPy <http://docs.plasmapy.org/en/latest/auto_examples/>`_
```

File: doc/projects_list.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
+=======================
+Who uses Sphinx-Gallery
+=======================
+
+Here is a list of projects using `sphinx-gallery`.
+
+* `Sphinx-Gallery <https://sphinx-gallery.github.io/en/latest/auto_examples/index.html>`_
```
Comment (choldgraf) [inline doc/projects_list.rst]:
```suggestion
* `Sphinx-Gallery <https://sphinx-gallery.github.io/auto_examples/index.html>`_
```

**Comment by emmanuelle** [comment]
I actually just copied the list from the README, so let me update this one as well.

File: doc/projects_list.rst (hunk 1 above)
Comment (emmanuelle) [reply doc/projects_list.rst]: It is SG : see https://github.com/pytorch/tutorials and https://github.com/pytorch/tutorials/blob/master/requirements.txt#L4 (too bad they pinned the version, though)

File: doc/projects_list.rst (hunk 2 above)
Comment (emmanuelle) [reply doc/projects_list.rst]: It's https://pyriemann.readthedocs.io/en/latest/index.html but the package does not seem to be updated any more, maybe we should just remove it.

**Comment by jarrodmillman** [comment]
You could add
```
`NetworkX <https://networkx.github.io/documentation/stable/auto_examples/index.html>`_
```

File: README.rst [hunk 5]
```diff
@@ -36,7 +36,7 @@ Who uses Sphinx-Gallery
 * `Cartopy <http://scitools.org.uk/cartopy/docs/latest/gallery/>`_
 * `PyVista <https://docs.pyvista.org/examples/>`_
 * `SimPEG <http://docs.simpeg.xyz/content/examples/>`_
-* `PlasmaPy <http://docs.plasmapy.org/en/master/auto_examples/>`_
+* `PlasmaPy <http://docs.plasmapy.org/en/latest/auto_examples/>`_
 * `Fury <http://fury.gl/latest/auto_examples/index.html>`_
```
Comment (choldgraf) [inline README.rst]:
```suggestion
* `Fury <http://fury.gl/latest/auto_examples/index.html>`_
* `NetworkX <https://networkx.github.io/documentation/stable/auto_examples/index.html>`_
```

File: doc/projects_list.rst [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+* `Astropy <http://docs.astropy.org/en/stable/generated/examples/index.html>`_
+* `SunPy <http://docs.sunpy.org/en/stable/generated/gallery/index.html>`_
+* `PySurfer <https://pysurfer.github.io/>`_
+* `Matplotlib <https://matplotlib.org/index.html>`_ `Examples <https://matplotlib.org/gallery/index.html>`_ and `Tutorials  <https://matplotlib.org/tutorials/index.html>`__
+* `PyTorch tutorials <http://pytorch.org/tutorials>`_
+* `Cartopy <http://scitools.org.uk/cartopy/docs/latest/gallery/>`_
+* `PyVista <https://docs.pyvista.org/examples/>`_
+* `SimPEG <http://docs.simpeg.xyz/content/examples/>`_
+* `PlasmaPy <http://docs.plasmapy.org/en/latest/auto_examples/>`_
+* `Fury <http://fury.gl/latest/auto_examples/index.html>`_
```
Comment (choldgraf) [inline doc/projects_list.rst]:
```suggestion
* `Fury <http://fury.gl/latest/auto_examples/index.html>`_
* `NetworkX <https://networkx.github.io/documentation/stable/auto_examples/index.html>`_
```

**Comment by choldgraf** [comment]
I restarted travis and he became happy, so I'm gonna merge this :-)

Thanks so much @emmanuelle !!!

**Pull request merged**